lib/src/chart/line_chart/line_chart_data.dart

```diff
@@ -21,13 +21,15 @@ class LineChartData extends AxisChartData {
   final FlTitlesData titlesData;
   final ExtraLinesData extraLinesData;
   final LineTouchData lineTouchData;
+  final RangeAnnotations rangeAnnotations;
```

Comment:
this field should be added to the lerp function.

Reply:
I'm not sure how to do this / what it does but I will try

Reply:
28ff618